firebase-storage/baseline.txt

```diff
@@ -0,0 +1,3 @@
+// Baseline format: 1.0
+RemovedInterface: com.google.firebase.storage.StorageException.ErrorCode:
+    Class com.google.firebase.storage.StorageException.ErrorCode no longer implements java.lang.annotation.Annotation
```

Comment:
Here as well, I wonder if api.txt is produced in a format that does not understand annotations? or some input flags are missing? might be worth asking metalava folks